tests/test_buffer_traditional.py

```diff
@@ -13,6 +13,7 @@ def setup_db(self, dvs):
         self.counter_db = dvs.get_counters_db()
 
         # enable PG watermark
+        self.config_db.update_entry('BUFFER_PG', 'Ethernet0|3-4', {'profile': 'ingress_lossless_profile'})
```

Comment:
do we still need this? in each test cases, lossless PG will be generated after `setup_teardown_test` and cable length being not `0m`.

Reply:
Agree, fixed.